CIP-0072/version_1_offchain.json

```diff
@@ -25,8 +30,8 @@
       "categories": {
         "type":"array",
         "items": {
-          "type": "string",
-          "enum":["Games", "DeFi", "Gambling", "Exchanges", "Collectibles", "Marketplaces", "Social", "Other"]
+          "type": "string", 
+          "enum": ["DeFi", "Development", "Education", "Games", "Identity", "Marketplace", "NFT", "Other", "Security"]
```

Comment:
What happens if an existing schema defines one of the categories that are being removed?  Does it not matter, given that prior schemas will not have `1` as a version number?  Or is this standard still in flux?  (I see it's still `Proposed` but your _Path to Active_ shows a couple implementations)

Thanks for catching me up here since I don't work with this tech directly.  If not deliberately kept as a Draft we can present it at the next CIP meeting (https://hackmd.io/@cip-editors/76) so I'd like to be clear on the context of these changes. 😇

Reply:
@rphair when publishing this CIP draft, we agreed the final category list might change due to ongoing UX research. 

Besides CRFA dApp Registrationand Certification service (CC: @matiwinnetou) and Lace DApp Store, both being early work in progress, we believe that there are no existing implementations of a DApp Store using this CIP. We're pretty confident about that, as we've not seen any on-chain activity (transactions with 1667 metadata label) since the CIP was published:

* [preview](https://preview.cexplorer.io/metadata/1667) - last transaction 6 months ago, before publishing this CIP, following legacy approach (CIP26)
* [preprod](https://preprod.cexplorer.io/metadata/1667) - last transaction 6 months ago, before publishing this CIP, following legacy approach (CIP26)
* [mainnet](https://cexplorer.io/metadata/1667) - no transactions using 1667 label

Therefore, we believe breaking changes at this point should be still OK, that's also why we left the following disclaimer in the **_Path to Active_** section:

> We reserve right to make small changes in this time, while the proposal is in the PROPOSED status

Moving forward, we'd like to suggest using the `version` property for any schema changes, even though it remains a draft.

Regarding CIP editors meeting, @matiwinnetou could you please help me here? I think this CIP has already been presented on one of those.

Reply:
thanks @marcin-mazurek - I'd actually missed the disclaimer comment & the further context is additionally helpful for confirmation.

We don't have to review these changes a CIP meeting before merging them (cc @matiwinnetou) although I've put in the Review section of the next meeting to make sure it keeps moving forward.

Reply:
Hey @rphair, 

Reading this comment thread. It seems that this one is already resolved/answered.